test/README.md

```diff
@@ -0,0 +1,60 @@
+# Testing
```

Comment:
I'd propose to generalise it, how people set it up is up to them.

Just list the requirements
- golang
- terraform
- aws-cli
- apparently eksctl?